{
  "contest": 164,
  "handle": "lucacez",
  "address": "0x20327e71a92aada0d2327cd455d748b5e11bea5c",
  "risk": "2",
  "title": "UNBOUNDED LOOPS MAY CAUSE `createClaimsBatch()` TO FAIL",
  "issueId": 464,
  "issueUrl": "https://github.com/code-423n4/2022-09-vtvl-findings/issues/464"
}